api/dimensions.go

```diff
@@ -15,6 +17,8 @@ import (
 	"github.com/gorilla/mux"
 )
 
+const maxIDs = 1000
```

Comment:
This probably needs to be a smaller number, as many browser URL length limits are 2k-4k and our codes are rarely only a few characters.

I'd suggest maybe ~200? Gives us some wiggle room for an average code size of 10 chars (which I think is probably over but will still cover us for geography)